pkg/controller/upgradeconfig/cluster_upgrader.go

```diff
@@ -548,7 +557,7 @@ func (cu ClusterUpgrader) UpgradeCluster(c client.Client, m maintenance.Maintena
 // check several things about the cluster and report problems
 // * critical alerts
 // * degraded operators (if there are critical alerts only)
-func performClusterHealthCheck(c client.Client) (bool, error) {
+func performClusterHealthCheck(c client.Client, upgradeConfig *upgradev1alpha1.UpgradeConfig) (bool, error) {
```

Comment:
Do we need to pass the entire UpgradeConfig object into this func when we only reference a string ? `upgradeConfig.Name`

Reply:
I am not sure how to pass the string to the func else, any suggestion?